lib/saml/user_attributes/id_me.rb

```diff
@@ -8,6 +8,15 @@ module UserAttributes
     class IdMe < Base
       include SentryLogging
       IDME_SERIALIZABLE_ATTRIBUTES = %i[first_name middle_name last_name zip gender ssn birth_date].freeze
+      LOA_MAPPING = {
```

Comment:
Would it be simpler to include this data in AUTHN_CONTEXTS?

Reply:
It won't map correctly, it's really only relevant to this class of user, but i'm open to suggestions.